controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java

```diff
@@ -67,7 +73,23 @@
 
     @Inject
     public CuratorDb(Curator curator) {
+        this(curator, curator.zooKeeperEnsembleConnectionSpec());
+    }
+
+    CuratorDb(Curator curator, String zooKeeperEnsembleConnectionSpec) {
         this.curator = curator;
+        this.zooKeeperEnsembleConnectionSpec = zooKeeperEnsembleConnectionSpec;
```

Comment:
I think it would be better to always get this string from the `Curator`, and instead use the `MockCurator.setZooKeeperEnsembleConnectionSpec(String s)` when needed.

Reply:
Fixed.